custom/icds_reports/views.py

```diff
@@ -1791,11 +1795,22 @@ def message(self, message_name):
             """
         return error_message_template.format(error_messages[message_name]).strip()
 
-    @method_decorator([api_auth, toggles.ICDS_DISHA_API_NIC.required_decorator()])
-    def get(self, request, *args, **kwargs):
+    def get_data(self, post_body):
+        xml_data = fromstring(post_body.strip())
```

Comment:
nit: it would be easier to read this code if the import was one level higher. `etree.fromstring` or `lxml.etree.fromstring` make it easier to understand this line. As it is i have no idea what we transforming the string to, without looking at the imports